lib/functions/rootfs/rootfs-create.sh

```diff
@@ -235,7 +235,11 @@ function create_new_rootfs_cache_via_debootstrap() {
 
 	# Remove `machine-id` (https://www.freedesktop.org/software/systemd/man/machine-id.html)
 	# Note: This will mark machine `firstboot`
-	run_host_command_logged echo "uninitialized" ">" "${SDCARD}/etc/machine-id"
+	# Note: As we don't use systemd-firstboot.service functionality, we make it empty to prevent services
+	# from starting up automatically on first boot on system version 2.50+. If someone is using the same,
+	# please reinitialize this to uninitialized. Do note that systemd will start all services then by
+	# default and that has to be handled in by setting system presets.
+	run_host_command_logged echo ">" "${SDCARD}/etc/machine-id"
```

Comment:
maybe `echo -n` to suppress unneeded newline would be a little cleaner?
systemd accepts both "empty" files though